src/Elastic.Apm/Model/Span.cs

```diff
@@ -38,6 +38,12 @@ internal class Span : ISpan
 		/// </summary>
 		private readonly StackFrame[] _stackFrames;
 
+		/// <summary>
+		/// Captures the sample rate of the agent when this span was created.
+		/// </summary>
+		[JsonProperty("sample_rate")]
+		internal readonly double SampleRate;
```

Comment:
```suggestion
		internal readonly double SampleRate { get; set; }
```

I think this should be a property rather than a field?

Reply:
Agreed - added a read-only property for this.